### ISSUE TITLE
Actionbar loading/animation issue

### ISSUE THREAD

**Issue by iMarwan-k** [NativeScript/NativeScript#5631]
### Did you verify this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?
Yes

### Tell us about the problem
Please, ensure your title is less than 63 characters long and starts with a capital
letter.
I have 2 pages the first one has `this.page.actionBarHidden = true;`, but the second one does have action bar. so when I go back and forth between then the actionbar doesn't behave appropriately 

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- CLI: (run `tns --version` to fetch it) 
3.4.3
- Cross-platform modules: (check the 'version' attribute in the
`node_modules/tns-core-modules/package.json` file in your project)
3.4.1
- Runtime(s): (look for the `"tns-android"` and `"tns-ios"` properties in the
`package.json` file of your project)
tns-ios : version:  3.4.1

- Plugin(s): (look for the version number in the `package.json` file of your
project)
"@angular/animations": "~5.2.0",
		"@angular/common": "~5.2.0",
		"@angular/compiler": "~5.2.0",
		"@angular/core": "~5.2.0",
		"@angular/forms": "~5.2.0",
		"@angular/http": "~5.2.0",
		"@angular/platform-browser": "~5.2.0",
		"@angular/platform-browser-dynamic": "~5.2.0",
		"@angular/router": "~5.2.0",
		"nativescript-angular": "~5.2.0",
		"nativescript-drop-down": "^3.2.4",
		"nativescript-gradient": "^2.0.1",
		"nativescript-iqkeyboardmanager": "^1.3.0",
		"nativescript-platform-css": "^1.6.5",
		"nativescript-theme-core": "~1.0.4",
		"reflect-metadata": "~0.1.8",
		"rxjs": "~5.5.2",
		"tns-core-modules": "~3.4.0",
		"zone.js": "~0.8.2"

### Please tell us how to recreate the issue in as much detail as possible.
![untitled](https://user-images.githubusercontent.com/19311396/38373322-a45791f8-38b6-11e8-856d-1dfb132e2f56.gif)

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
first page
`<Gradient direction="to bottom" colors="#003E74, #003E74, #003E74, #001F3A">
    
    <GridLayout id="webIdGridTextFields" class="gridWithTextField" rows="auto">
        <TextField class="textField" row="0" hint="Organization Web ID" autocorrect="false"></TextField>
    </GridLayout>

    <GridLayout id="webIdBtnGrid" class="gridWithBtnAtButtom m-t-10" rows="auto,auto,auto" horizontalAlignment="center">
        <Button id="submitBtn" class="primaryBtn" text="Submit" row="0" (tap)="submit()"></Button>
    </GridLayout>
</Gradient> `

second page
```
<ActionBar class="action-bar">
    <Label text="Hello" textWrap="true"></Label>
</ActionBar>
    <GridLayout id="userAndPassBtnGrid" class="gridWithBtnAtButtom" rows="auto" horizontalAlignment="center">
        <Button id="nextBtn" class="primaryBtn" row="0" text="Next" (tap)="next()"></Button>
    </GridLayout>
</Gradient>
```

**Comment by iMarwan-k** [comment]
Any comment or respond on this

**Comment by NickIliev** [comment]
@iMarwan-k sorry for the delay in our answer,
This was indeed a real issue fixed with the latest release. I've tested the issue with the latest tns-core-modules (version 4.0.0) and the issue is resolved on my side (tested with iOS 11.3 and iPhone 8).

This is [the test application used ](https://github.com/NickIliev/NS-Issues-2018/tree/master/NativeScript/issue_5631)- please do let me know if you are still experiencing this issue with the latest NativeScript and if so on what device and iOS version.

**Comment by NickIliev** [comment]
Closing this one as resolved but please do let m,e know if you are still experiencing any similar issues with a different code base.
